client/src/components/Navbar/navbar.css

```diff
@@ -21,8 +21,8 @@
 
 .dropdownMenuBlock {
   position: absolute;
-  top: 2.5rem;
-  right: 0;
```

Comment:
Without the arrow, I like this offset a little bit at 0. Especially if we're not keeping the down arrow.

Reply:
The dropdown was covering the avatar a bit. I think it looks good if it shows just below it

Reply:
That's fair enough. If it was overlapping that's fair enough